src/scripts/ci/setup_travis.sh

```diff
@@ -74,5 +74,5 @@ if [ "$TRAVIS_OS_NAME" = "linux" ]; then
     fi
 
 elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
-    HOMEBREW_NO_AUTO_UPDATE=1 brew install ccache
+    HOMEBREW_NO_AUTO_UPDATE=1 brew install ccache boost
```

Comment:
i believe this proved unnecessary, didn't it?

Reply:
true. un-cherry-picked.